### PR TITLE
Slot icons micro-refactor

### PR DESCRIPTION
**Pull request by Core0verload**
A tiny bit of HUD code is refactored.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Neat.

**Comment by Fox-McCloud** [comment]
I'm not a big fan of this, personally, @Core0verload 

For new players, it's pretty helpful to know which items go in which slot, not to mention far less confusing. SS13 is a pretty daunting game to new players---now it's going to be even more confusing to tell them where to put things.

"Where do I put my  gloves?"

"oh, the slot that you open up on the left most slot that's the slot just below the middle one to the most right slot".

As opposed to "click on the backpack slot icon, then click the gloves no the one that has a picture of gloves".

**Comment by Core0verload** [comment]
@Fox-McCloud, these icons still show up when there is no item. They are hidden only when there are actual gloves in gloves slot.

**Comment by Fox-McCloud** [comment]
@Core0verload 

Ahh, I see. Well, that's not so bad then.

**Comment by RemieRichards** [comment]
:-1: I don't like it.

**Comment by TrustyGun** [comment]
I don't like this either. It looks weird. :-1:

**Comment by Core0verload** [comment]
It looks weird because we got used to icons underneath the items, and maybe because some items are off-center. When I was a new player, icons under items were looking weird to me.

**Comment by Giacom** [comment]
How about instead of getting rid of the icon entirely, you just make it transparent? I think that would look better.

**Comment by optimumtact** [comment]
>small hud code tweak
>25 files changed

why is hud code so ass

**Comment by TankNut** [comment]
:-1: Definitely not a fan. Without the placeholders the inventory slots look a lot more bland and things like the headset end up looking weird.

**Comment by Archie700** [comment]
Can we put it to a toggle in setting

**Comment by YotaXP** [comment]
I like it. :+1: The icons are pretty useless when they're obstructed, and only make it harder to identify the item in the slot.

Though I find it funny that the headset in that slot makes the ear icon look sort of like a head.

**Comment by ChangelingRain** [comment]
The icons in the slot are needed if only to make the UI look decent; it looks really bland and flat when the icons are removed even if the lack of icon is partially obscured by the item in the slot

**Comment by Core0verload** [comment]
If the amount of negative feedback will not reduce, I will just remove the placeholders change and rename it to "Slot icons micro-refactor".

**Comment by Cheridan** [comment]
while we're all throwing around subjective opinions i'll say that i like it

**Comment by PKPenguin321** [comment]
>If the amount of negative feedback will not reduce, I will just remove the placeholders change and rename PR to "Slot icons micro-refactor".

nah just change the title to refactor then remove the comparison images. maybe say it reduces lag or something. bam, ez merge.

**Comment by ChangelingRain** [comment]
If you do that I will close the PR.

**Comment by PKPenguin321** [comment]
if we were serious we wouldnt post that in a thread with multiple maintainers in it changelingrain jeez it's a joke

**Comment by Core0verload** [comment]
I converted this thing to refactor, just like I promised. The only icon that still changes it's state is a backpack icon. Because it's always obscured by backpack anyway.

**Comment by ChangelingRain** [comment]
Don't do it to backpacks either some people actually use satchels

**Comment by duncathan** [comment]
I use satchels exclusively

**Comment by ChangelingRain** [comment]
yes some people(all the maintainers probably) use satchels

**Comment by RemieRichards** [comment]
I also use a satchel, and am a maintainer.
revert it for backpacks too.

**Comment by Fox-McCloud** [comment]
I def use a satchel...

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -27,19 +28,107 @@
...
+/obj/screen/inventory/hand
+	var/image/active_overlay
+	var/image/handcuff_overlay
 
+/obj/screen/inventory/hand/update_icon()
+	..()
+	if(!active_overlay)
+		active_overlay = image("icon"=icon, "icon_state"="hand_active")
+	if(!handcuff_overlay)
+		var/state = (slot_id == slot_r_hand) ? "markus" : "gabrielle"
```
Comment (Iamgoofball) [inline code/_onclick/hud/screen_objects.dm]: uh

Comment (RemieRichards) [inline code/_onclick/hud/screen_objects.dm]: they've always been called this.

**Comment by Core0verload** [comment]
It looks even better with a satchel than with a backpack. But if you really want...

**Pull request merged**